content/learn/using-validated-pattern-operator-adoc.adoc

```diff
@@ -0,0 +1,95 @@
+---
+menu:
+  learn:
+    parent: Infrastructure
+title: Using the Validated Patterns Operator (asciidoc)
```

Comment:
Drop the '(asciidoc)' before we merge it? Or do you want to kill the old md file first and then drop it?

Reply:
Hi @mbaldessari. Yes, I'll drop the asciidoc before I merge it and delete the old file (.md) at the same time.